test-lib.sh

```diff
@@ -307,8 +307,10 @@ ct_path_foreach ()
 function ct_run_test_list() {
   for test_case in $TEST_LIST; do
     : "Running test $test_case"
-    [ -f test/$test_case ] && source test/$test_case
-    [ -f ../test/$test_case ] && source ../test/$test_case
+    # shellcheck source=/dev/null
+    [ -f test/"$test_case" ] && source test/"$test_case"
```

Comment:
```suggestion
    [ -f "test/$test_case" ] && source "test/$test_case"
```

Reply:
Suggestion: put the project-preferences to guidelines document.

In this case, both variants are entirely correct, and I for one usually
prefer the former variant.